README.md

```diff
@@ -45,16 +45,19 @@ Represent an individual build of a package.
 
 Column             | Type             | Summary
 ------------------ | ---------------- | ------------
-build_id (pk)      |  text            | A build's unique id
-previous_build_id  |  text            | Hold a reference to the previous build id
-rollback_build_ids |  map<text, text> | timestamp string mapped to the rollback id
-env                |  text            | What enviroment is this build made for (dev, test, etc.)
-name               |  text            | What package has been built
-version            |  text            | What version of a package does this build represent
-fingerprints       |  set<text>       | Primary keys for `build_files`, represents the unique contents of the file
-artifacts          |  set<text>       | fingerprint/file-name
-recommended        |  set<text>       | Possible reduced set of artifacts based
-a build's configuration
+env (pk)           | text             | What environment is this build made for (dev, test, etc.)
+name (pk)          | text             | What package has been built
+version (pk)       | text             | What version of a package does this build represent
+build_id           | text             | A build's unique id
+previous_build_id  | text             | Hold a reference to the previous build id
+rollback_build_ids | map<text, text>  | timestamp string mapped to the rollback id
+locale (ck)        | text             | What locale this was built for
+create_date        | timestamp        | Time of creation
+value              | text             | ???
```

Comment:
all database tables have a value.. don't you know :) 

![](https://media.giphy.com/media/NCTbhL8AG2s8g/giphy.gif)

Reply:
But to be fair :) I lack context as well as to why that field exists.

Reply:
I would feel bad about leaving the `?`s here, but there are already some in other places :/ 

This is where we add rows for this table: https://github.com/warehouseai/bffs/blob/2f139501665d6a7e929150d750de63375b497202/index.js#L386 `value` isn't in there so it's likely unused, but since we're creating the column, I'd rather document it.